applications/tari_console_wallet/src/ui/state/wallet_event_monitor.rs

```diff
@@ -113,6 +121,22 @@ impl WalletEventMonitor {
                         trace!(target: LOG_TARGET, "Wallet Event Monitor received wallet connectivity status changed");
                         self.trigger_peer_state_refresh().await;
                     },
+                    Ok(_) = software_update_notif.changed() => {
+                        trace!(target: LOG_TARGET, "Wallet Event Monitor received wallet auto update status changed");
+                        let update:Option<SoftwareUpdate> = match *software_update_notif.borrow() {
```

Comment:
The type annotation can be inferred 

```suggestion
                        let update = match *software_update_notif.borrow() {
```

Reply:
I've change the code, but there has to be a local variable anyway. Even the compiler suggestion is `help: consider moving this into a 'let' binding to create a shorter lived borrow`